test/lib/analyzer/binaries-analyzer.test.ts

```diff
@@ -109,6 +109,19 @@ test('analyze', async t => {
         { name: 'openjdk-jre', version: '1.8.0_191-b12' },
       ],
     },
+    {
+      description: 'adoptOpenJDK present in image',
+      targetImage: 'adoptopenjdk/openjdk10',
+      binariesOutputLines: { node: '',
+                             openjdk: `Picked up JAVA_TOOL_OPTIONS: -XX:+UseContainerSupport
+                                       openjdk 10.0.2-adoptopenjdk 2018-07-17
+                                       OpenJDK Runtime Environment (build 10.0.2-adoptopenjdk+13)
+                                       OpenJDK 64-Bit Server VM (build 10.0.2-adoptopenjdk+13, mixed mode)` },
+      installedPackages: [],
+      expectedBinaries: [
+        { name: 'openjdk-jre', version: '10.0.2-adoptopenjdk+13' },
```

Comment:
the expected version should be `10.0.2`

Reply:
I understand that it still qualifies as plain opendjk and versions should ideally match, but even opendjk 10.0.2 returns a more elaborate version:

Leons-MacBook-Pro:snyk-docker-plugin leongold$ docker run -it openjdk:10.0.2 /bin/sh
# java --version
openjdk 10.0.2 2018-07-17
OpenJDK Runtime Environment (build 10.0.2+13-Debian-2)
OpenJDK 64-Bit Server VM (build 10.0.2+13-Debian-2, mixed mode)

Looking at other fixtures, we don't confine versions to strict x.y.z either, e.g. "1.8.0_191-b12"

Reply:
We can resolve this, right @orkamara? (decided to keep `adoptopenjdk` in version here and use the translator)